examples/programmatic/scripts/start.js

```diff
@@ -0,0 +1,8 @@
+const { loadNuxt } = require('nuxt-start')
```

Comment:
Should we expose `loadNuxt` in `nuxt-start` ? If so, is this `loadNuxt` only for `start` ?

Reply:
`loadNuxt` is for both `nuxt` and `nuxt-start`, for example, on serverless function where you only want to load nuxt for production, you only include `nuxt-start` to reduce the lambda size

Reply:
I understand it’s same api, but as it’s imported from nuxt-start, so for:’start’ makes me feel confusing if it’s duplicate and also if I can put any other value here.

Reply:
I see what you mean, actually we also have the `'dry'` value that could work for `nuxt-start`